src/main/views/alphabetical-search.njk

```diff
@@ -74,5 +75,5 @@
 
 {% endblock %}
 {% block bodyEnd %}
-  <script src="assets/js/back-to-top.js"></script>
+  <script src="alphabetical-dev.js"></script>
```

Comment:
did you need to change it here too or is this something else?

Reply:
nah this one needs to stay as this (the frontend loads the bundle, which loads the back-to-top)

Reply:
ok brill, as long as everything runs 👍🏼